configure.ac

```diff
@@ -1222,10 +1190,8 @@ AC_CHECK_DECLS([fchmodat], [], [], [[#define _GNU_SOURCE 1
                                     #include <sys/stat.h>]])
 AC_CHECK_DECLS([readlinkat], [], [], [[#define _GNU_SOURCE 1
                                        #include <unistd.h>]])
-AC_REPLACE_FUNCS(openat fstatat fchownat fchmodat readlinkat)
 
 AC_CHECK_DECLS([log2], [], [], [[#include <math.h>]])
-AC_REPLACE_FUNCS(log2)
```

Comment:
@vpodzime can we do some cleanup of these checks in core, nova, enterprise as well? How did you go about knowing which ones to remove?

Reply:
Oops. I thought this was a libntech PR ;) this is core. Awesome that you removed all those checks. 🥇